dfindexeddb/indexeddb/cli.py

```diff
@@ -73,14 +75,19 @@ def _Output(structure, output):
   elif output == 'repr':
     print(structure)
 
-
```

Comment:
Currently using pylint but it doesn't seem to be flagging blank line issues.  I will see if I can update that as a separate PR as it may flag lint issues elsewhere.

Per https://google.github.io/styleguide/pyguide.html#35-blank-lines, there should be two blank lines between top level function/class definitions.  Seems I used it inconsistently so I have updated to 2.  Good catch thanks.